src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncIteratorTests.cs

```diff
@@ -3120,19 +3120,17 @@ .maxstack  3
 }", sequencePoints: "C+<M>d__0.MoveNext", source: source);
         }
 
-        [ConditionalFact(typeof(WindowsDesktopOnly), Reason = ConditionalSkipReason.NativePdbRequiresDesktop, AlwaysSkip = "PROTOTYPE(local-function-attributes)")]
+        [ConditionalFact(typeof(WindowsDesktopOnly), Reason = ConditionalSkipReason.NativePdbRequiresDesktop)]
         public void AsyncIteratorWithAwaitCompletedAndYield_WithEnumeratorCancellation_LocalFunction()
         {
             string source = @"
 using System.Runtime.CompilerServices;
-using System.Threading.Tasks;
 using System.Threading;
 class C
 {
-    async Task M()
+    void M()
     {
-        await foreach (var i in local(default)) { }
-
+#pragma warning disable 8321 // Unreferenced local function
         async System.Collections.Generic.IAsyncEnumerable<int> local([EnumeratorCancellation] CancellationToken token)
```

Comment:
Looking at [PRs that implemented EnumeratorCancellation support](https://github.com/dotnet/roslyn/pull/39978), I see that there are a few scenarios that are interesting and should probably be verified on local functions as well:
- see `WRN_UndecoratedCancellationTokenParameter`
- see `WRN_UnconsumedEnumeratorCancellationAttributeUsage`

I'm not sure if those checks will trigger on local functions at the moment.